CHANGELOG.md

```diff
@@ -1,8 +1,20 @@
 # CHANGELOG
-## Release 126.0.0 (in development)
+## Release 126.0.0-alpha.3
 
 ### Enhancements
 
+- Salt Master worker_threads and timeout are now configurable
+  (PR[#4149](https://github.com/scality/metalk8s/pull/4149))
+
+- Alert re-push mechanism works properly
+  (PR[#4141](https://github.com/scality/metalk8s/pull/4141))
+
+- Versions is properly shown on UI after upgrade
+  (PR[#4140](https://github.com/scality/metalk8s/pull/4140))
+
+- Prevent Python 3.6 conflicts with newer versions of OpenStackSDK
+  (PR[#4139](https://github.com/scality/metalk8s/pull/4139))
+
```

Comment:
This line should be removed it's not supposed to be show to the user, it's only for CI

Reply:
Done !!!